package.json

```diff
@@ -2,14 +2,19 @@
   "name": "iopipe",
   "version": "0.3.1",
   "description": "IOpipe Lambda Analytics & Tracing Agent",
-  "main": "index.js",
+  "main": "dist/iopipe.js",
   "scripts": {
-    "test": "istanbul cover jasmine --captureExceptions"
+    "test": "istanbul cover jasmine --captureExceptions",
+    "build": "webpack --progress --profile --colors",
+    "prepublish": "webpack --progress --profile --colors"
```

Comment:
I think this needs to be `prepublishOnly` because when I try to do `npm install` on my machine just to get the deps, it goes through this step but errors because it doesn't have deps yet